plugins/optimization-detective/storage/post-type.php

```diff
@@ -190,7 +190,7 @@ static function ( OD_URL_Metric $url_metric ): array {
 			},
 			$group_collection->get_flattened_url_metrics()
 		),
-		JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES // TODO: No need for pretty-printing.
+		JSON_UNESCAPED_SLASHES // No need for escaped slashes since not printed to frontend.
```

Comment:
Good question. The slashes should always be present when the JSON is inline in HTML, as this prevents malicious strings from breaking out of the containing `script`. For example, if someone somehow got `</script>` into the data being serialized to JSON, this would prematurely terminate the `script` element, whereas if slashes are escaped, then `<\/script>` would not.

In the context of `post_content` here where the JSON is never being printed to HTML, there is no need for this escaping.

Reply:
Er, so that should probably be restored to Speculation Rules 😊

Reply:
I see, thanks for explaining. I believe Speculation Rules already does this correctly then, feel free to check. I was referring to the Make Core post draft I've been working on.